controller/errors.js

```diff
@@ -0,0 +1,14 @@
+exports.res404 = {error: "<strong>Error 404</strong> File Not Found"};
+
+exports.fileNotFound = (req, res) => {
+    // Error 404 Handler
+    res.status(404).send({error: "<strong>Error 404</strong> File Not Found"});
+};
+
+exports.testNext404 = (err, req, res, next) => {
+    res.status(404).send({error: "<strong>Error 404</strong> File Not Found"});
```

Comment:
I understand the motivation here but these errors are sent as JSON. They will not interpreted as HTML by the client. I would remove any tags here and handle how you want errors to be displayed in the front end.